plugins/cryptolib/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ include_directories (
          ${HPCC_SOURCE_DIR}/system/include
          ${HPCC_SOURCE_DIR}/system/jlib
          ${HPCC_SOURCE_DIR}/rtl/include
+         ${HPCC_SOURCE_DIR}/dali/base
```

Comment:
These dependencies don't look right.  Why are these now required?

Reply:
@ghalliday These dependencies were introduced with adding the ICodeContext to the plugin API call